helpers/deploymentConfig.ts

```diff
@@ -160,6 +164,13 @@ export const ADDRESSES: PreconfiguredAddresses = {
     eBTC_Accountant: "0x1b293DC39F94157fA0D1D36d7e0090C8B8B8c13F",
     pufETH: "0xD9A442856C234a39a81a089C06451EBAa4306a72",
     LBTC: "0x8236a87084f8B84306f72007F36F2618A5634494",
+    newPTOracle: "0x9a9fa8338dd5e5b2188006f1cd2ef26d921650c2",
```

Comment:
Do we need to keep both?

Reply:
https://github.com/VenusProtocol/oracle/pull/243/commits/299d4fd460fe74727e2246801542f7b30ef15de5